api/v1alpha1/dataprotectionapplication_types.go

```diff
@@ -357,6 +359,45 @@ type NodeAgentCommonFields struct {
 	PodConfig *PodConfig `json:"podConfig,omitempty"`
 }
 
+type RestorePVC struct {
+	// IgnoreDelayBinding indicates to ignore delay binding the restorePVC when it is in WaitForFirstConsumer mode
+	IgnoreDelayBinding bool `json:"ignoreDelayBinding,omitempty"`
+}
+
+type RuledConfigs struct {
+	// NodeSelector specifies the label selector to match nodes
+	NodeSelector metav1.LabelSelector `json:"nodeSelector"`
+
+	// Number specifies the number value associated to the matched nodes
+	Number int `json:"number"`
+}
+
+// LoadConcurrency is the config for data path load concurrency per node.
+type LoadConcurrency struct {
+	// GlobalConfig specifies the concurrency number to all nodes for which per-node config is not specified
+	GlobalConfig int `json:"globalConfig,omitempty"`
+
+	// PerNodeConfig specifies the concurrency number to nodes matched by rules
+	PerNodeConfig []RuledConfigs `json:"perNodeConfig,omitempty"`
+}
+
+type NodeAgentConfigMapSettings struct {
+	// LoadConcurrency is the config for data path load concurrency per node.
+	// +optional
+	LoadConcurrency *LoadConcurrency `json:"loadConcurrency,omitempty"`
+	// BackupPVCConfig is the config for backupPVC (intermediate PVC) of snapshot data movement
+	// +optional
+	BackupPVCConfig map[string]nodeagent.BackupPVC `json:"backupPVC,omitempty"`
+	// RestoreVCConfig is the config for restorePVC (intermediate PVC) of generic restore
+	// +optional
+	RestorePVCConfig *RestorePVC `json:"restorePVC,omitempty"`
+	// PodResources is the resource config for various types of pods launched by node-agent, i.e., data mover pods.
+	// +optional
+	PodResources *kube.PodResources `json:"podResources,omitempty"`
+	// LoadAffinity is not required within NodeAgentConfigMapSettings,
```

Comment:
@sseago added comment here as well.

Reply:
Hmm, thinking about this, shouldn't keep all the same options as upstream ?

Reply:
I think @mpryc is trying to avoid exposing more than one place to set.

We can keep as a dummy one or perhaps as a higher priority one?